spec/injection_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'injection'
+
+describe Array do
+
+  describe 'injection' do
+    subject{[1,2,3]}
+
+    it 'should take subject and block and return the value of the block ' do
+      expect(subject.injection{|x,y| x + y}).to eq 6
+    end
+
+    it 'should take subject and block and return the value of the block' do
+      expect(subject.injection{ |x,y| x + (2*y) }).to eq 12
```

Comment:
Space missing after comma.<br>Surrounding space missing for operator '*'.<br>Space missing to the left of {.